src/botpress.js

```diff
@@ -235,8 +232,8 @@ class botpress {
       db,
       emails,
       mediator,
-      convo,
-      umm,
+      renderers,
+      umm: renderers, // DEPRECATED
```

Comment:
Good. You can also go fancy and replace it with getter property that will return `renderers` but also log the deprecation warning to console.

Reply:
@emirotin Done, excellent idea 👍